challenge.md

```diff
@@ -31,20 +31,20 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
+ <!-- Answer Starts Here --> It is an absolute path. It shows where the file originated.
+
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
- 
+ <!-- Answer Starts Here --> We would use an absolute path because we are not not using the same directory.
+
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
- <!-- Answer Starts Here -->
- 
+ <!-- Answer Starts Here --> we would use a relative path to get it. It is within the same directory.
```

Comment:
We use a relative path because we want our links within the project to be preserved, _even if the project folder gets moved_. If we used links with Absolute paths, those links would break the minute we moved our project folder.